src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +34,18 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL:
+                return String.valueOf(LocalDate.now());
```

Comment:
we user LocalDate.now() four times here, let's make it a local variable inside this method